openverse_catalog/dags/providers/provider_api_scripts/nappy.py

```diff
@@ -0,0 +1,111 @@
+"""
+Content Provider:       Nappy
+
+ETL Process:            Use the API to identify all CC0-licensed images.
+
+Output:                 TSV file containing the image meta-data.
+
+Notes:                  This api was written specially for Openverse.
+                        There are no known limits or restrictions.
+
+"""
+import logging
+
+from common import constants
+from common.licenses import get_license_info
+from common.loader import provider_details as prov
+from providers.provider_api_scripts.provider_data_ingester import ProviderDataIngester
+
+
+logger = logging.getLogger(__name__)
+
+
+class NappyDataIngester(ProviderDataIngester):
+    providers = {"image": prov.NAPPY_DEFAULT_PROVIDER}
+    endpoint = "https://api.nappy.co/v1/openverse/images"
+    # TODO The following are set to their default values. Remove them if the defaults
+    # are acceptible, or override them.
+    delay = 1
+    retries = 3
+    headers = {"Accept": "application/json"}
```

Comment:
Should we set an Openverse user-agent by default in `ProviderDataIngester`? That might be nice to standardize. I'll add it here shortly as well.